cmd/create.go

```diff
@@ -155,8 +156,13 @@ A full list of component types that can be deployed is available using: 'odo cat
 			fmt.Printf("Triggering build from %s.\n\n", componentGit)
 
 			// Git is the only one using BuildConfig since we need to retrieve the git
-			err = component.Build(client, componentName, applicationName, true, true, stdout)
+			err = component.Build(client, componentName, applicationName, componentWatch, componentWatch, stdout)
 			odoutil.CheckError(err, "")
+			// if watch flag is not present
+			if !componentWatch {
+				fmt.Printf("This may take few moments to be ready\n")
+				return
```

Comment:
I think there is no reason to return here. We should let the control flow and display the component related info at the end. Also I think the above print statement is unnecessary too.